action.yml

```
@@ -48,7 +51,12 @@ runs:
         JULIA_PKG_SERVER: ""
     - run: |
         # The Julia command that will be executed
-        julia_cmd=( julia --check-bounds=yes --color=yes --depwarn=${{ inputs.depwarn }} --inline=${{ inputs.inline }} --project=${{ inputs.project }} -e 'import Pkg;include(joinpath(ENV["GITHUB_ACTION_PATH"], "kwargs.jl"));kwargs = Kwargs.kwargs(;coverage = :(${{ inputs.coverage }}),force_latest_compatible_version = :(${{ inputs.force_latest_compatible_version }}),);Pkg.test(; kwargs...)' )
+        check_bounds_opt=' '
+        if [[ ${{ inputs.check_bounds }} != 'auto' ]]
+        then
+          check_bounds_opt='--check-bounds=${{ inputs.check_bounds }} '
+        fi
```

Sorry for causing that error, bash quotes are hard 🤦 

```suggestion
        check_bounds_opt=''
        if [[ ${{ inputs.check_bounds }} != 'auto' ]]
        then
          check_bounds_opt='--check-bounds=${{ inputs.check_bounds }}'
        fi
```

Maybe this? If not, I'm fine with reverting it to what it was before since that didn't fail the build.